packages/open-ui/src/components/Tag/Tag.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import { MarginStyleKeys } from 'interfaces/MarginStyleKeys';
+import { getMarginProps } from 'utils/helpers';
+
+import styles from './Tag.module.scss';
+
+interface TagProps {
+  name: string;
+  size?: 'small' | 'large';
+}
+
+export const Tag: React.FC<TagProps & MarginStyleKeys> = props => {
+  const { name, size = 'small' } = props;
+  const _props = { marginBottom: 0.5, marginRight: 0.5, ...props };
```

Comment:
I dont think there should be any default margins on *any* components. Lets keep it clean so we can use them as we need in `client`

Reply:
@remotealex the only reason I put em was to not include the marginStyles everytime we gonna use them as I felt we gonna use them in form of list. But yeah I agree to not have default margins